backend/tests/test_llm_model_validation.py

```
@@ -0,0 +1,80 @@
+    [
+        (models.CaptureRefineRequest, "model_size"),
+        (models.LLMGenerateRequest, "model_size"),
+        (models.CaptureSettingsResponse, "llm_model"),
+        (models.CaptureSettingsUpdate, "llm_model"),
+    ],
+)
+def test_rejects_unknown_model_name(model_cls, field):
+    kwargs = {field: "not-a-real-model"}
+    if model_cls is models.LLMGenerateRequest:
+        kwargs["prompt"] = "hi"
+    with pytest.raises(Exception):
```

Ruff (0.16.2)

[warning] Do not assert blind exception: `Exception`

(B017)